app/client/templates/single-app/single-app.html

```diff
@@ -50,7 +50,7 @@
                   <div class="key">Ported by:</div><div class="value">{{author.name}}&nbsp;</div>
                   <div class="key">License:</div><div class="value">{{verboseNone license}}&nbsp;</div>
                   <div class="key">Code:</div><div class="value">{{{codeFormat codeLink}}}&nbsp;</div>
-                  <div class="key">Version:</div><div class="value" data-tooltip={{versionChanges}} data-tooltip-direction="s">{{version}}&nbsp;</div>
+                  <div class="key">Version:</div><div class="value" data-tooltip={{renderedChangelog}} data-tooltip-direction="s">{{version}}&nbsp;</div>
```

Comment:
With this change, the client is now dependent on the api to send back a "reasonable" changelog value. If an app has many versions, the tooltip will overflow. Please create an issue to confirm the behaviour of the field.